src/build.py

```diff
@@ -825,6 +825,15 @@ def OverrideCMakeCompiler():
           '-DCMAKE_CXX_COMPILER=' + CXX]
 
 
+def CMakeCommand(args):
+  command = [PREBUILT_CMAKE_BIN, '-G', 'Ninja'] + args
+  # Python's location could change, so always update CMake's cache
+  command.append('-DPYTHON_EXECUTABLE=%s' % proc.Which('python', os.getcwd()))
```

Comment:
This should probably just be `sys.executable` so everything is always guaranteed to use the same python.